### PR TITLE
Adds 'Chicken' Nuggets

### PR DESCRIPTION
**Pull request by TehZombehz**
Adds 'chicken' nuggets\* in 4 shapes, including regular, star, corgi, and lizard.
They're essentially just lumps of cooked meat.
Tablecrafting a nugget will result in one of the 4 possible shapes at random.
 -*not made of actual chicken, but cooked meat cutlets
nuggets
nuggets2
Let me know if the sprites need some fixing. Like true 'shaped' chicken nuggets, they're only supposed to vaguely look like their desired shapes.

### CODE REVIEW

**Comment by Cheridan** [comment]
adorable.

you know we have actual chickens, right? ...this way is probably funnier though

**Comment by TehZombehz** [comment]
I decided to keep the recipe simple enough that you don't have to go hunting for a chicken somehow, otherwise these will rarely be made. It also kinda reflects the questionable origin of all those mass-produced chicken nuggets.

**Comment by bgobandit** [comment]
made with real corgi/lizard

**Comment by Nienhaus** [comment]
The lizard makes me think of Reptar from Rugrats.

**Comment by KorPhaeron** [comment]
Amazing

File: code/modules/food&drinks/food/snacks_meat.dm [hunk 1]
```diff
@@ -220,3 +220,28 @@
...
 	list_reagents = list("nutriment" = 6, "capsaicin" = 5)
 	filling_color = "#FA8072"
+
+/obj/item/weapon/reagent_containers/food/snacks/nugget
+	name = "chicken nugget"
+	filling_color = "#B22222"
+	bonus_reagents = list("nutriment" = 1, "vitamin" = 1)
+	list_reagents = list("nutriment" = 2)
+
+/obj/item/weapon/reagent_containers/food/snacks/nugget/New()
```
Comment (KorPhaeron) [inline code/modules/food&drinks/food/snacks_meat.dm]: When you do New(), you gotta call the parent with 

..() 

before the rest of this

**Comment by Ausops** [comment]
i can see a butt on the first one but I will be a good boy for these either way

**Comment by KorPhaeron** [comment]
@Iamgoofball 

Need to finish your economic system so we can begin saving gbp

**Comment by Zilenan91** [comment]
The only map you can reliably get and breed chickens on is Metastation since there's one in the garden.

**Comment by Archie700** [comment]
Those aren't actually made from chicken though.

**Comment by optimumtact** [comment]
What happened to goof animal husbandry?

**Comment by KorPhaeron** [comment]
@TehZombehz 

I rewrote it to have less copypaste, you will need to change the icon name in the dmi to nugget_lump though

http://pastebin.com/B41U9aJi

**Comment by Cheridan** [comment]
why closed

**Comment by KorPhaeron** [comment]
@Cheridan 

Git problems, will be reopened soon

File: tgstation.dme [hunk 2]
```diff
@@ -6,6 +6,78 @@
...
+#define FILE_DIR "sound/magic"
+#define FILE_DIR "sound/mecha"
+#define FILE_DIR "sound/misc"
+#define FILE_DIR "sound/piano"
+#define FILE_DIR "sound/spookoween"
+#define FILE_DIR "sound/violin"
+#define FILE_DIR "sound/voice"
+#define FILE_DIR "sound/voice/complionator"
+#define FILE_DIR "sound/vox_fem"
+#define FILE_DIR "sound/weapons"
```
Comment (duncathan) [inline tgstation.dme]: Uh oh

**Pull request closed without merging**